sys/net/link_layer/Kconfig.lorawan

```
@@ -0,0 +1,483 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config HAVE_LORAWAN
+    bool
+    help
+        Indicates that LoRaWAN headers are present.
+
+menuconfig KCONFIG_USEMODULE_LORAWAN
+    bool "Configure LoRaWAN MAC"
+    depends on USEMODULE_GNRC_LORAWAN || USEPKG_SEMTECH_LORAMAC || USEMODULE_RN2XX3
```

Then here the symbol above should be used right?
```suggestion
    depends on HAVE_LORAWAN
```

---

Yes. I had a bit of confusion with this. Shall DM you.